defaults/main.yml

```diff
@@ -6,6 +6,7 @@ solr_user: solr
 
 solr_version: "6.6.0"
 solr_mirror: "https://archive.apache.org/dist"
+solr_package: "{{ solr_mirror }}/lucene/solr/{{ solr_version }}/{{ solr_filename }}.tgz"
```

Comment:
We will use this to download the `.tgz` from a Maven repo instead of an Apache mirror.